lib/VeresOne.js

```diff
@@ -1,36 +1,42 @@
 /*!
  * Copyright (c) 2018-2019 Veres One Project. All rights reserved.
  */
-'use strict';
 
+/**
+ * Veres One has 3 modes dev, test, and live
+ * @typedef {'live'|'test'|'dev'} Mode
+ */
+
+'use strict';
 const constants = require('./constants');
 const documentLoader = require('./documentLoader');
 const VeresOneDidDoc = require('./VeresOneDidDoc');
 const storage = require('./storage');
 const {LDKeyPair} = require('crypto-ld');
 const VeresOneClient = require('./VeresOneClient');
 const VeresOneClientError = require('./VeresOneClientError');
-
 const jsonld = require('jsonld');
 const jsigs = require('jsonld-signatures');
 
 class VeresOne {
   /**
-   * @param [options={}] {object}
-   *
-   * @param [options.mode='test'] {string} Ledger mode ('test', 'dev', 'live'),
-   *   determines hostname for ledger client.
-   * @param [options.hostname] {string} Optional hostname override. If not
+   * VeresOne contains the over all settings for your
+   * Ledger, keystore, and other parts of the block-chain.
+   *
+   * @param {Object} [options={}] - Options setup the Veres class.
+   * @param {Mode} [options.mode='test'] -
+   * Ledger mode 'test', 'dev', 'live',
+   * determines hostname for ledger client.
+   * @param {string} [options.hostname] - Optional hostname override. If not
    *   provided, ledger hostname will be determined based on `mode`.
-   *
-   * @param [options.logger] {object} Optional logger (defaults to console)
-   * @param [options.client] {WebLedgerClient}
+   * @param {Object} [options.logger] - Optional logger (defaults to console).
+   * @param {WebLedgerClient} [options.client] - A WebLedgerClient.
    *
    * Storage defaults to file-based stores, can be substituted to in-memory
    * for testing.
-   * @param [options.keyStore] {Store} Private key store
-   * @param [options.didStore] {Store} Local DID Doc cache
-   * @param [options.metaStore] {Store} DID Doc metadata store
+   * @param {Store} [options.keyStore] - Private key store.
+   * @param {Store} [options.didStore] - Local DID Doc cache.
+   * @param {Store} [options.metaStore] -  DID Doc metadata store.
```

Comment:
extra space after hyphen.

Reply:
are you sure you want no space after the hyphen? this is eslint-plugin-jsdoc's default fix behavior which was able to correct 20+ issues in one fix. space - space is more readable and let's us use --fix which results in quicker linter. less time spent on docs etc.

Reply:
@aljones15 are you looking at the diff on github?  On line 39 there are two spaces after the hyphen that occurs before `Did Doc metatadata store.`, there should only be one space.

Reply:
yeah I guess fix does not check that. will correct soonish.